CONTRIBUTING.md

```diff
@@ -626,9 +624,40 @@ To install DocFX on a Microsoft Windows machine:
 > [!TIP]
 > Alternative ways to install DocFX can be found on the [DocFX website](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool).
 
-#### Making a test build
+#### First-time setup
+
+Before you make your first test build, we recommend the first-time setup detailed below. If you do not do this, you will still be able to make a test build, but there may be many warnings in the terminal, and some pages may not be included in the build.
+
+You only need to do this setup once.
+
+1. Check if your fork is up to date with the latest changes from the main dataminer-docs repository (See [Make sure your fork is up to date](#make-sure-your-fork-is-up-to-date).
+
+1. Make sure you have Visual Studio 2017 or higher installed.
+
+1. Download and install [Build Tools](https://aka.ms/vs/17/release/vs_BuildTools.exe).
+
+1. Go to the folder containing your dataminer-docs repository.
+
+   > [!TIP]
+   > To quickly find this folder from Visual Studio Code, right-click a folder from the repository in the Explorer tab and select *Reveal in File Explorer*.
 
-When you have finished making a series of changes to documentation in Visual Studio Code, do the following to make a test build on your local machine.
+   1. In the `\dataminer-docs\src\DataMiner\` subfolder open the folder *Dataminer.sln* with Visual Studio.
+
+   1. Go to the solution explorer in the top-right corner, right-click the solution node and select *Restore NuGet Packages*.
+
+      ![Restore Nuget Packages](~/user-guide/images/Contributing_Solution_Node.png)
+
+   1. Start a build by pressing CTR+SHIFT+b.
+
+   1. Repeat the steps above for the file *Code Library.sln* in the `dataminer-docs\src\Base Class Library\` subfolder of the repository.
+
+1. Open Visual Studio Code and go to *Terminal* > *New Terminal*.
+
+1. In the terminal, enter `docfx metadata -f`.
+
+   If everything is OK, you will get no warnings.
+
```

Comment:
Note that the need for building the solution is due to the change we had to make in the DocFX file regarding how the API docs is generated. Now it is generating the metadata from the DLL/XML. Previously, it was via the source code, which does not require a build (The long term goal is to be able to again generate it via the source code when that issue is fixed). Note that this also means that whenever there was an update to the code/documentation in the solutions, these solutions will need to be rebuilt again.

Reply:
I've made a couple of changes based on your feedback. Could you take another look?